дз1ахлюстина/дз1ахлюстина/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace дз1ахлюстина
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double x;
+            Console.WriteLine("Введите длину ребра октаэдра:");
+            x = double.Parse(Console.ReadLine());
+
+            double S;
```

Comment:
Замечание о стиле: локальные переменные принято назвать с маленькой буквы.
```suggestion
            double s;
```
Еще лучше, если имя будет значимым
```suggestion
            double square;
```